Cargo.toml

```
@@ -81,10 +130,11 @@ strip = "symbols"
 
 [features]
 default = ["memory-cache"]
-dhat-heap = ["dep:dhat"] 
+dhat-heap = ["dep:dhat"]
 memory-cache = ["dep:mini-moka"]
-redis-cache = ["dep:redis","dep:base64"]
-compress-cache-results = ["dep:async-compression","dep:cfg-if"]
-encrypt-cache-results = ["dep:chacha20poly1305","dep:chacha20"]
-cec-cache-results = ["compress-cache-results","encrypt-cache-results"]
+redis-cache = ["dep:redis", "dep:base64"]
+compress-cache-results = ["dep:async-compression", "dep:cfg-if"]
+encrypt-cache-results = ["dep:chacha20poly1305", "dep:chacha20"]
+cec-cache-results = ["compress-cache-results", "encrypt-cache-results"]
 experimental-io-uring = ["actix-web/experimental-io-uring"]
+use-synonyms-search = ["dep:thesaurus"]
```

```suggestion
use-synonyms-search = ["dep:thesaurus\static"]
use-non-static-synonyms-search = ["dep:thesaurus"]
```

I would suggest having two separate features for static and non-static synonyms search as this will allow devices with low-memory to also take advantage of the feature but also using lesser memory usage. :slightly_smiling_face: .